etl/transform.py

```diff
@@ -90,15 +89,46 @@ def transform(self) -> DataFrame:
                 "source",
             )
         )
-        return with_recommendations.union(without_recommendations)
+
```

Comment:
I wonder if we could get away with using a `F.when(F.col('instanceof').isNull).then(..).otherwise(..)` statement instead of splitting into two queries. It would be nice to avoid doing the join later on.

Reply:
Done